src/Microsoft.AspNetCore.WebHooks.Receivers.Slack/SlackCommand.cs

```diff
@@ -74,14 +74,19 @@ public static class SlackCommand
         /// <see cref="KeyValuePair{StringSegment, StringSegment}.Value"/> from what
         /// <see cref="ParseActionWithValue"/> returned.
         /// </param>
+        /// <param name="errorMessage">
+        /// A <see cref="string"/> describing the specific problem when an issue is encountered. <see langword="null"/>
+        /// when the method successfully returns a non-<see langword="null"/> dictionary.
+        /// <paramref name="errorMessage"/> and the return value are never both <see langword="null"/>.
+        /// </param>
         /// <returns>
-        /// A <see cref="ValueTuple{T1, T2}"/> in which one property is always <see langword="null"/>. When successful,
-        /// <c>Error</c> is <see langword="null"/> and <c>Parameters</c> an <see cref="IDictionary{TKey, TValue}"/>
-        /// mapping zero or more parameter names to zero or more values. Otherwise, <c>Parameters</c> is
-        /// <see langword="null"/> and <c>Error</c> a <see cref="string"/> describing the specific problem.
+        /// An <see cref="IDictionary{TKey, TValue}"/> mapping zero or more parameter names to zero or more values.
+        /// <see langword="null"/> when an issue is encountered. <paramref name="errorMessage"/> and the return value
+        /// are never both <see langword="null"/>.
         /// </returns>
-        public static (IDictionary<StringSegment, IList<StringSegment>> Parameters, string Error) ParseParameters(
-            StringSegment text)
+        public static IDictionary<StringSegment, IList<StringSegment>> TryParseParameters(
```

Comment:
consider converting this to a `bool TryParseParameters(text, out paramters, out errorMessage)`

I usually expect a bool when I see `Try`. 

I don't think it's super important

Reply:
Went with your not "super important" leaning, partially because the double-`out` part looks icky and partially to avoid redoing the doc comments yet again.